docs/hardware-interfaces/pinouts.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Pinouts
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This article explains the layout and available connection options for Omega2/2+ and Omega2S/Omega2S+ devices. It covers pinout information, including hardware interfaces, to provide a clear understanding of the pin layout structure.
+
+Let's get started 🚀
```

Comment:
Instead of this `Let's get started 🚀`, please add a header called Device Pinouts